### DISCUSSION TITLE
FileNotFoundError

### DISCUSSION THREAD

**Discussion by Mafiiou** [openai/whisper#1448] [General]
I always make the same mistake, I don't understand 

FileNotFoundError                         Traceback (most recent call last)
Cell In[50], line 20
     17 import whisper
     19 model = whisper.load_model("large")
---> 20 result = model.transcribe(chemin_absolu)
     21 print(result["text"])

File [~\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\whisper\transcribe.py:121](https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/daslv/Desktop/M%C3%A9linda/~/AppData/Local/Packages/PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0/LocalCache/local-packages/Python310/site-packages/whisper/transcribe.py:121), in transcribe(model, audio, verbose, temperature, compression_ratio_threshold, logprob_threshold, no_speech_threshold, condition_on_previous_text, initial_prompt, word_timestamps, prepend_punctuations, append_punctuations, **decode_options)
    118     decode_options["fp16"] = False
    120 # Pad 30-seconds of silence to the input audio, for slicing
--> 121 mel = log_mel_spectrogram(audio, padding=N_SAMPLES)
    122 content_frames = mel.shape[-1] - N_FRAMES
    124 if decode_options.get("language", None) is None:

File [~\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\whisper\audio.py:130](https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/daslv/Desktop/M%C3%A9linda/~/AppData/Local/Packages/PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0/LocalCache/local-packages/Python310/site-packages/whisper/audio.py:130), in log_mel_spectrogram(audio, n_mels, padding, device)
    128 if not torch.is_tensor(audio):
    129     if isinstance(audio, str):
--> 130         audio = load_audio(audio)
    131     audio = torch.from_numpy(audio)
    133 if device is not None:

File [~\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\whisper\audio.py:46](https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/daslv/Desktop/M%C3%A9linda/~/AppData/Local/Packages/PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0/LocalCache/local-packages/Python310/site-packages/whisper/audio.py:46), in load_audio(file, sr)
     27 """
     28 Open an audio file and read as mono waveform, resampling as necessary
...
   1471     self._close_pipe_fds(p2cread, p2cwrite,
   1472                          c2pread, c2pwrite,
   1473                          errread, errwrite)

FileNotFoundError: [WinError 2] Le fichier spécifié est introuvable

**Comment by phineas-pta** [comment]
can you compare your file path in windows vs `chemin_absolu` ? it shouldn't be too long nor contain special characters
